.github/stale.yml

```diff
@@ -0,0 +1,19 @@
+only: issues
+daysUntilStale: 30
+daysUntilClose: 7
+exemptLabels:
+  - pinned
+  - security
+  - feature-request
+  - enhancement
+  - to-be-reproduced
+  - bug
+  - question
```

Comment:
should question be exempt? We have lots of questions that are pending-response and we never get more information. Seems like those should be auto-closed after a while